script.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.btn');
+const navMenu = document.querySelector('.menu-sec');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.menu-sec').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```

Comment:
You did well with your mobile popup, kindly implement the following changes so as to enable you have a better design and user experience 😄 :

- According to the requirement, when a user clicks any of the mobile menu options , it should direct you to that part of the page. For example, when I click on the `About` it takes me to the about section, but when I click on `Contact` it doesn't take me to the contact section and also, the `Portfolio` should take me to the project section 

Please kindly note that the background for the popup menu is blur in the Figma design given, kindly implement this for your project 😄 

Figma design :
![image](https://github.com/mar4444/re-myportfolio/assets/20537712/9b57efcd-841c-42b9-b5d8-780d5f81a1eb)

Your design :
![image](https://github.com/mar4444/re-myportfolio/assets/20537712/6ab8c140-c592-4eb5-826f-0a13b67fd126)